packages/renderer/src/lib/pod/PodActions.svelte

```diff
@@ -119,6 +142,18 @@ if (dropdownMenu) {
       menu="{dropdownMenu}"
       detailed="{detailed}"
       icon="{faRocket}" />
+    {#if openingUrls !== undefined}
+      {#each openingUrls as url}
+        <ListItemButtonIcon
+          title="Open Browser"
+          onClick="{() => window.openExternal(url)}"
```

Comment:
from my understanding user will see "Open Browser" several times if I have like 4 URLs
but how to differentiate each link if I only see "Open Browser"

also does it mean we will have like 5 icons if I have 5 links ?